lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -113,6 +114,25 @@ def set_score(self, json_data, suffix):  # pylint: disable=unused-argument
         )
 
 
+class CompletableStatelessXBlock(XBlock):
```

Comment:
What does "Stateless" refer to in this context?  I'm confused by it, so not sure yet if it is appropriate or not.  Maybe the doc string will clarify.

Reply:
I'm not sure, I was copypasting from the Graded one.  How about "StubCompletableXBlock," since it's a testing stub with no real functionality.